lib/resources/r4/person.yaml

```diff
@@ -0,0 +1,27 @@
+---
+name: Person
+field_name_base_url: https://hl7.org/fhir/R4/person-definitions.html#Person
+fields:
+- name: identifier
+  type: List of Identifier
+  description: The identifier for the person.
+  action: terminology
+  children:
+
+  - name: type
+    type: CodeableConcept
+    description: The type for the identifier that can be used to determine which identifier to use for a specific purpose.
+    url: https://hl7.org/fhir/r4/datatypes-definitions.html#Identifier.type
+    action: terminology
+    binding:
+      description: The type used to determine which identifier to use for a specific purpose.
+      terminology:
+      - display: Identifier Type Codes
```

Comment:
This can be removed

Reply:
removed in commit f6e7999c7e7fb613eeacc38ecfe1f445a4e9480d